Documentation/ci-nnstreamer/README.md

```diff
@@ -0,0 +1,13 @@
+# How to use config files
+
+These are TAOS-CI files for nnstreamer.
+
+- http://suprem.sec.samsung.net/STAR/TAOS-CI
+
+## How to use
+
+After adding TAOS-CI submodule in your project, copy and overwrite them.
+
+```bash
+$ yes | cp -rf $PWD/Documentation/taos/* $PWD/TAOS-CI/ci/taos/
```

Comment:
I think that you keep "yes" command to handle the existing files  of  ./ci/taos/***.  Is it right? Looks good.